### PR TITLE
Update eqMac cask

### PR DESCRIPTION
**Pull request by nodeful**
Updating [bitgapp/eqMac](https://github.com/bitgapp/eqMac) cask to pull the latest version from official website: https://eqmac.app

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

### CODE REVIEW

**Comment by nodeful** [comment]
The CI won't pass because it's trying to install a macOS audio driver and failing... 
How to get around this?

**Comment by gigajosh** [comment]
@nodeful, you could put a message in the output to guide the user to open System Prefs and allow installation of the driver, then run the install again. This is what you need to do when installing eqMac manually from the dmg anyway.

**Comment by nodeful** [comment]
Hmm, but there's no need to open System Preferences to install the driver. Just a Sudo shell script that puts file in the correct System folder and restarts Core Audio. The cask will work on users Mac, as is, just the CI doesn't pass.

**Comment by jstader** [comment]
> Hmm, but there's no need to open System Preferences to install the driver. Just a Sudo shell script that puts file in the correct System folder and restarts Core Audio. The cask will work on users Mac, as is, just the CI doesn't pass.

The previous [merge](https://github.com/Homebrew/homebrew-cask/pull/80657) was allowed with the same error since the Travis CI job passed.

File: Casks/eqmac.rb [hunk 1]
```diff
@@ -1,26 +1,20 @@
...
-  app "eqMac#{version.major}.app"
+  app 'eqMac.app'
   installer script: {
-                      executable: "#{staged_path}/eqMac#{version.major}.app/Contents/Resources/install_driver.sh",
+                      executable: "#{staged_path}/eqMac.app/Contents/Resources/install_driver.sh",
                       sudo:       true,
                     }
 
-  uninstall quit:   "com.bitgapp.eqMac#{version.major}",
+  uninstall quit:   'com.bitgapp.eqmac',
```
Comment (miccal) [inline Casks/eqmac.rb:15]:
```
uninstall quit:   [
                           'com.bitgapp.eqMac',
                           'com.bitgapp.eqmac-LaunchAtLoginHelper',
                           'com.bitgapp.eqmac.driver',
                         ],

**Comment by miccal** [comment]
@nodeful let's try my suggested changes and see how it goes.

File: Casks/eqmac.rb [hunk 2]
```diff
@@ -1,26 +1,20 @@
 cask 'eqmac' do
-  version '2.2'
-  sha256 'ff60579197b52571d9686bbfaec695dfa75797b8a9314fbf69a673bbeb12e1fc'
+  version :latest
+  sha256 :no_check
```
[the remark this answers is not in the document]
Comment (nodeful) [reply Casks/eqmac.rb:3]: I don't really like pinning to a specific version as I don't have the time to update the hash on every release (there will be loads soon). I understand the security implication if someone takes hold of my domain however. I'm using Sparkle framework to update the app itself through an appcast, but it's different from the one Cask uses. Basically, is there a way to automate updating of the cask file with the latest version and the hash? Maybe by using GitHub releases?

File: Casks/eqmac.rb (hunk 1 above)
Comment (nodeful) [reply Casks/eqmac.rb:15]: Yeah quitting the app and LaunchAtLogin Helper is a good call, I will add that in. No need to quit the driver however, it's not an executable :)

**Comment by nodeful** [comment]
I've added these lines to the install/uninstall scripts:
https://github.com/bitgapp/eqMac/blob/d0b2d2bc312f4cfa88ca5a0a493c737962fec672/native/app/Source/Scripts/install_driver.sh#L3-L7
It's on the development brach at the moment, but I will include that in the next release. This should skip installation of the Driver during CI.
Need to resolve other comments about versioning + hash first though.

File: Casks/eqmac.rb [hunk 3]
```diff
@@ -1,26 +1,20 @@
...
 
-  url "https://github.com/bitgapp/eqMac/releases/download/V#{version}/eqMac#{version.major}.dmg"
-  appcast 'https://github.com/bitgapp/eqMac/releases.atom'
+  url 'https://update.eqmac.app/eqMac.dmg'
```
[the remark this answers is not in the document]
Comment (nodeful) [reply Casks/eqmac.rb:5]: Sorry for my noob questions I don't use Cask that much, but my users demand distribution through it. Does the releases.atom appcast automatically update the cask?

[the remark this answers is not in the document]
Comment (miccal) [reply Casks/eqmac.rb]: No, but [updating a cask is very easy](https://github.com/Homebrew/homebrew-cask/blob/fc0a1d0633cd9c9cd04e6928be9a0e50bb43944b/CONTRIBUTING.md#updating-a-cask).

File: Casks/eqmac.rb [hunk 4]
```diff
@@ -1,26 +1,30 @@
 cask 'eqmac' do
-  version '2.2'
-  sha256 'ff60579197b52571d9686bbfaec695dfa75797b8a9314fbf69a673bbeb12e1fc'
+  version '0.0.1'
+  sha256 '00538e305f4ad90c688fc4fd9dd07969f713e35051ef98971cda53368996f918'
 
-  url "https://github.com/bitgapp/eqMac/releases/download/V#{version}/eqMac#{version.major}.dmg"
+  # github.com/bitgapp/eqMac was verified as official when first introduced to the cask
```
Comment (miccal) [inline Casks/eqmac.rb:5]:
```
# github.com/bitgapp/eqMac/ was verified as official when first introduced to the cask
```
The trailing slash is [there for a reason](https://github.com/Homebrew/homebrew-cask/pull/80988).

**Comment by miccal** [comment]
@nodeful the fact that the Travis CI passes is I think a good indication that everything is fine, but a maintainer will need to provide a final review and merge this.

File: Casks/eqmac.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (miccal) [reply Casks/eqmac.rb]: Cask prefers `version`'ed submissions, especially if they have GitHub releases.

File: Casks/eqmac.rb (hunk 3 above)
[the remark this answers is not in the document]
Comment (nodeful) [reply Casks/eqmac.rb]: That's a shame the documentation permits your to use :latest without telling your your PR will be blocked 😄 I would like to have as little work to do as possible as I'm just one man developing the app. Hopefully users will be updating the cask for me...

**Comment by nodeful** [comment]
I'm going to release v0.1.0 today and all CI's should pass since I will skip driver installation. But feel free to merge this one regardless

File: Casks/eqmac.rb (hunk 3 above)
[the remark this answers is not in the document]
Comment (miccal) [reply Casks/eqmac.rb]: I am sure they will -- you can see that I have linked an issue and another pull request to your pull request, indicating interest from users.

File: Casks/eqmac.rb (hunk 4 above)
Comment (nodeful) [reply Casks/eqmac.rb]: Fixed! Excuse the mistake

**Comment by miccal** [comment]
@nodeful one of the Travis CI jobs passed, so all good I think :)

**Comment by nodeful** [comment]
@miccal no something is wrong. Travis CI says install_driver.sh not found. Looking into it...

**Comment by nodeful** [comment]
Hmm odd...

**Comment by miccal** [comment]
Travis CI runs two jobs, and one passed:

Screen Shot 2020-04-28 at 19 34 51

**Comment by nodeful** [comment]
Right I see. Please sorry for the confusion 🙏

**Comment by miccal** [comment]
No problem at all :)

**Pull request merged**